### ISSUE TITLE
Compare checksums instead of sizes when checking outputs

### ISSUE THREAD

**Issue by annawoodard** [matz-e/lobster#168]
Both of the tools we use to check stageout transfers (xrdfs and Chirp) have functionality to conveniently calculate checksums. Should we use these to check that output files have been transferred correctly, or is comparing the file size sufficient?

**Comment by matz-e** [comment]
It's expensive, but depending on how expensive, it may be worth it do this properly with checksums.

**Comment by annawoodard** [comment]
We already need to calculate it once for publishing, so we get it for half price. I don't think this amount of worker side CPU matters much to us. I only mention it because there's been some pain in CRAB3 land over related issues lately.

**Comment by matz-e** [comment]
Good point!  Then I think we should always calculate the checksum on the worker, check it with the SE if possible and save us the work when publishing.
